SimPEG/electromagnetics/time_domain/simulation_1d_stitched.py

```diff
@@ -0,0 +1,208 @@
+import numpy as np
+from scipy import sparse as sp
+from ... import utils
+from ..base_1d_stitched import BaseStitchedEM1DSimulation
+from .simulation_1d import Simulation1DLayered
+from .survey import Survey
+from ... import maps
+from multiprocessing import Pool
+
+
+def run_simulation_time_domain(args):
+    import os
+
+    os.environ["MKL_NUM_THREADS"] = "1"
+    """
+    This method simulates the EM response or computes the sensitivities for
+    a single sounding. The method allows for parallelization of
+    the stitched 1D problem.
+    :param src: a EM1DTM source object
+    :param topo: Topographic location (x, y, z)
+    :param np.array thicknesses: np.array(N-1,) layer thicknesses for a single sounding
+    :param np.array sigma: np.array(N,) layer conductivities for a single sounding
+    :param np.array eta: np.array(N,) intrinsic chargeabilities for a single sounding
+    :param np.array tau: np.array(N,) Cole-Cole time constant for a single sounding
+    :param np.array c: np.array(N,) Cole-Cole frequency distribution constant for a single sounding
+    :param np.array chi: np.array(N,) magnetic susceptibility for a single sounding
+    :param np.array dchi: np.array(N,) DC susceptibility for magnetic viscosity for a single sounding
+    :param np.array tau1: np.array(N,) lower time-relaxation constant for magnetic viscosity for a single sounding
+    :param np.array tau2: np.array(N,) upper time-relaxation constant for magnetic viscosity for a single sounding
+    :param float h: source height for a single sounding
+    :param string output_type: "response", "sensitivity_sigma", "sensitivity_height"
+    :param bool invert_height: boolean switch for inverting for source height
+    :return: response or sensitivities
+    """
+
+    (
+        source_list,
+        topo,
+        thicknesses,
+        sigma,
+        eta,
+        tau,
+        c,
+        chi,
+        dchi,
+        tau1,
+        tau2,
+        h,
+        output_type,
+        # return_projection,
+        # coefficients
+    ) = args
+
+    n_layer = len(thicknesses) + 1
+    n_src = len(source_list)
+
+    local_survey = Survey(source_list)
+    wires = maps.Wires(("sigma", n_layer), ("h", n_src))
+    sigma_map = wires.sigma
+    h_map = wires.h
+
+    sim = Simulation1DLayered(
+        survey=local_survey,
+        thicknesses=thicknesses,
+        sigmaMap=sigma_map,
+        hMap=h_map,
+        eta=eta,
+        tau=tau,
+        c=c,
+        topo=topo,
+        hankel_filter="key_101_2009",
```

Comment:
Should probably be passed as a keyword with default.

Reply:
again open to any suggestions.